package.json

```diff
@@ -22,6 +22,15 @@
     "build-production-client": "NODE_ENV=production BABEL_ENV=production yarn build-client",
     "build-production": "yarn distclean && yarn && gulp languages:extract && yarn build-production-client",
     "build-languages": "gulp languages",
+    "docker:up": "docker-compose -f docker/docker-compose.yml up",
+    "docker:stop": "docker-compose -f docker/docker-compose.yml stop",
+    "docker:down": "docker-compose -f docker/docker-compose.yml down",
+    "docker:build": "docker-compose -f docker/docker-compose.yml build",
+    "docker:cli": "docker container exec jetpack_wordpress wp --allow-root --path=/var/www/",
```

Comment:
you should use `docker-compose run/exec` here because the name prior to `_wordress` is generated from the folder name, so this would break if I did `git clone https://jetpack jetpack-testing` because docker-compose would create `jetpack-testing_wordpress` containers.

Reply:
I believe @enejb you changed phpstorm command to this format to be able to pass args from yarn to Docker?

Reply:
It will actually still work I guess because we've fixed the container name to `jetpack_wordpress` in the yml here: https://github.com/Automattic/jetpack/pull/8949/files#diff-072e81fe88d9b23fe00fc189279936a6R40

Reply:
I've just tried to switch to `docker-compose exec wordpress wp --allow-root --path=/var/www/` and it does not work (`ERROR: No container found for wordpress_1`). 

Trying with  `docker-compose exec jetpack_wordpress wp --allow-root --path=/var/www/` fails because docker-compose expects a service name instead of a container name (`ERROR: No such service: jetpack_wordpress`).